<?xml version="1.0" encoding="utf-8"?>
<launch>
  <arg name="global_env_pcd2_topic" value="/random_forest/all_map" />
  <arg name="map_size_x" value="50.0" />
  <arg name="map_size_y" value="50.0" />
  <arg name="map_size_z" value="8.0" />
  <arg name="origin_x" value=" -25.0" />
  <arg name="origin_y" value=" -25.0" />
  <arg name="origin_z" value=" -1.0" />
  <arg name="resolution" value="0.5" />

  <arg name="steer_length" value="2.0" />
  <arg name="search_radius" value="6.0" />
  <arg name="search_time" value="0.2" />
  <arg name="max_tree_node_nums" value="5000" />

  <node pkg="path_finder" type="path_finder" name="path_finder_node" output="screen">
    <remap from="/global_cloud" to="$(arg global_env_pcd2_topic)"/>

    <param name="occ_map/origin_x" value="$(arg origin_x)" type="double"/>
    <param name="occ_map/origin_y" value="$(arg origin_y)" type="double"/>
    <param name="occ_map/origin_z" value="$(arg origin_z)" type="double"/>
    <param name="occ_map/map_size_x" value="$(arg map_size_x)" type="double"/>
    <param name="occ_map/map_size_y" value="$(arg map_size_y)" type="double"/>
    <param name="occ_map/map_size_z" value="$(arg map_size_z)" type="double"/>
    <param name="occ_map/resolution" value="$(arg resolution)" type="double"/>

    <param name="RRT_Star/steer_length" value="$(arg steer_length)" type="double"/>
    <param name="RRT_Star/search_radius" value="$(arg search_radius)" type="double"/>
    <param name="RRT_Star/search_time" value="$(arg search_time)" type="double"/>
    <param name="RRT_Star/max_tree_node_nums" value="$(arg max_tree_node_nums)" type="int"/>

  </node>

</launch>